### PR TITLE
Use time.ParseDuration error message

### PR DESCRIPTION
**Pull request by andrewkroh**

### CODE REVIEW

File: error_test.go [hunk 1]
```diff
@@ -56,6 +58,9 @@ func TestErrorMessages(t *testing.T) {
 		"arr_oob_nested_wo_meta": raiseIndexOutOfBounds(arrNested, 5),
 		"arr_oob_nested_w_meta":  raiseIndexOutOfBounds(arrNestedMeta, 5),
 
+		"invalid_duration": raiseInvalidDuration(newString(
```
Comment (andrewkroh) [inline error_test.go]: I wasn't sure what the naming convention was here.

Comment (urso) [inline error_test.go]: testMeta is adding the `(source: ...)` to the message. Maybe introduce 2 tests (`invalid_duration_wo_meta` and `invalid_duration_w_meta`)

**Comment by andrewkroh** [comment]
I'll pick this back up next week. I kind of forgot about it, sorry.

File: error_test.go (hunk 1 above)
Comment (andrewkroh) [inline error_test.go]: I added the test case as suggested and renamed the existing one.

File: testdata/error/message/invalid_duration.golden [hunk 2]
```diff
@@ -0,0 +1 @@
+time: unknown unit  hour in duration 1 hour accessing 'timeout' (source:'test.source')
```
Comment (urso) [inline testdata/error/message/invalid_duration.golden]: This golden file has become obsolete. Maybe -update shall delete old golden files before (another PR)

Comment (andrewkroh) [inline testdata/error/message/invalid_duration.golden]: I update the PR to remove that file.

**Pull request merged**